### PR TITLE
[kf5service] new port

### PR DESCRIPTION
**Pull request by wrobelda**
**Describe the pull request**

- #### What does your PR fix?  
  Adds a new kf5service port. Depends on:
  - #19179. Will rebase once it gets merged. 
  - #20016 

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  All official are supported.

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

**Comment by JonLiu1993** [comment]
@wrobelda ,If this pr is ready, please click ready for review

**Comment by wrobelda** [comment]
> @wrobelda ,If this pr is ready, please click ready for review

Not ready yet, depends on 2 other PRs. Will let you know.

**Review by JonLiu1993 — CHANGES_REQUESTED** [review]

File: ports/kf5coreaddons/portfile.cmake [hunk 1]
```diff
@@ -3,41 +3,34 @@ vcpkg_from_github(
     REPO KDE/kcoreaddons
     REF v5.84.0
     SHA512 58a802d03bea135c192265651540f9f87066b20ee4af620a94a874ec3992c5e734d1ab1e4ccb082de97389fa3479fd3672ec586c3dd36c9b4a5422c7be98d119
+    PATCHES
+        122.patch # https://invent.kde.org/frameworks/kcoreaddons/-/merge_requests/122
+        129.patch # https://invent.kde.org/frameworks/kcoreaddons/-/merge_requests/129
```
Comment (JonLiu1993) [inline ports/kf5coreaddons/portfile.cmake:8]: Could you please rename these patch files to briefly describe what you are fixing?

File: ports/kf5crash/portfile.cmake [hunk 2]
```diff
@@ -4,25 +4,28 @@ vcpkg_from_github(
     REF v5.84.0
     SHA512 db37a481e0d34e0024cabd77c22680171fa383ea300c56c73f807380fd818263d27f36fb012fc6604b462dbf9374db39bab94c4dc2c3bff488ac9812d01eb2dd
     HEAD_REF master
+    PATCHES
+        23.patch # https://invent.kde.org/frameworks/kcrash/-/merge_requests/23
```
Comment (JonLiu1993) [inline ports/kf5crash/portfile.cmake:8]: Could you please rename these patch files to briefly describe what you are fixing?

**Comment by wrobelda** [comment]
@JonLiu1993 this no longer depends on other PR and is ready for a final review.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JonLiu1993** [comment]
@wrobelda ,Thanks for your harding working! LGTM

**Comment by wrobelda** [comment]
> @wrobelda ,Thanks for your harding working! LGTM

@JonLiu1993 this now depends on #20016. See https://github.com/microsoft/vcpkg/pull/19195#discussion_r703106264 for discussion.

**Comment by JonLiu1993** [comment]
depends on pr #20006

**Comment by wrobelda** [comment]
> depends on pr #20006

@JonLiu1993 surely you meant #20016? It's been merged to master and I rebased this PR, so it's good to go.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the ports :D

**Pull request merged**